packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -84,21 +88,14 @@ export class RemoteMessageProcessor {
 	}
 }
 
-const copy = (remoteMessage: ISequencedDocumentMessage): ISequencedDocumentMessage => {
-	// Do shallow copy of message, as the processing flow will modify it.
-	// There might be multiple container instances receiving same message
-	// We do not need to make deep copy, as each layer will just replace message.content itself,
-	// but would not modify contents details
-	const message = { ...remoteMessage };
```

Comment:
@vladsud - Please take a look.  Navigating through the code I believe `remoteMessage` is the shallow copy from [ContainerRuntime.process](https://github.com/microsoft/FluidFramework/pull/17280/files#diff-d6dd5b7a4910beea419ad4e2e60191c88283bac177f1d87b9046be1592a763feR2162) above in the PR (where I moved the comment to)

Reply:
I brought this up in [original PR 13273](https://github.com/microsoft/FluidFramework/pull/13273/files#r1322228455) (after the fact). My understanding is that it is duplicated for defense in depth. Didn't get an answer to how accurate the comment really is.

Reply:
I think we can defend removing it.  I'm marking the class as `@internal` (it's not publicly exported), renaming the variable to `remoteMessageCopy`, and adding commentary in the function's doc comment about how it's a shallow copy.

Given that this function has a single callsite and literally the previous line to the call site copies this, I don't think we need this defense-in-depth.

Reply:
Btw I don't see the test coverage @andre4i mentioned that would be validating that it's a shallow copy (and no tests failed)

Reply:
The test checking what the remote processor does with shallow copies should be `packages/runtime/container-runtime/src/test/opLifecycle/remoteMessageProcessor.spec.ts:"Always processing a shallow copy of the message"`.

Reply:
Ok my "Find-all-usages" was busted in VS Code.  Thanks.  Interestingly enough, that test passes even w/o the shallow copy here.  I'm stepping through to see where the output obj is created, I just went through it once and missed it this time, pretty subtle!

Reply:
Ok it's this `stamp` function:

<img width="309" alt="image" src="https://github.com/microsoft/FluidFramework/assets/12305068/db042976-fb0c-41a0-866a-9c43e60e0317">

Reply:
Looping back here from an offline conversation - Andrei opened AB#5494 about the test infra creating a shallow copy.  We agreed that this copy is extraneous, and I'm prepared to take full responsibility if something goes sideways from its removal :)